### PR TITLE
Drewr/outage points

### PR DESCRIPTION
**Pull request by drewmresnick**
working on displaying outage data as point layer on map

### CODE REVIEW

File: PRISM_maproom/layout.py [hunk 1]
```diff
@@ -66,6 +67,15 @@
 )
 
 
+#point_to_layer = assign("""function(feature, latlng, context){
+#    style = {
+#        radius: 5,
+#        zIndex: 1000,
+#        color: 'black',
+#        opacity: 0.7,};
```
Comment (aaron-kaplan) [inline PRISM_maproom/layout.py:75]: I finally figured out how to make the CircleMarkers stay on top even after mouseover events. Add to this `style` object the following properties:
```
pane: 'markerPane',
interactive: false
```
`pane` makes it stay on top, and `interactive` makes mousing over the marker behave the same as mousing over the polygon below it.

BTW this object should really be called `options`, not `style`. It's not a list of CSS style properties. See [doc](https://leafletjs.com/reference.html#circlemarker).

**Comment by remicousin** [comment]
Taking over as Drew might be offline for some part of the day.

**Comment by drewmresnick** [comment]
i think i have gotten it to a point that I am satisfied with for today. let me know if you see anything glaring

**Comment by aaron-kaplan** [comment]
If it works then it's good enough for a demo. If you want to merge to
facilitate collaboration with Rémi, feel free. Keep track of my comments
somewhere so you can address them later.

**Comment by remicousin** [comment]
ah ok... I will stop messing around then

**Review by remicousin — APPROVED** [review]

**Comment by drewmresnick** [comment]
@remicousin if you prefer to mess around on this branch go ahead, I won't be making any more edits to it now. Just polishing the poster at this point. So it is up to you if you would like to keep playing around on this branch or not.

**Pull request merged**